components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/OpcNodeOnEndpointModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿// ------------------------------------------------------------
```

Comment:
can't we just use OpcNodeModel instead of thisone?

Reply:
removed